sec-keys/sigstore-trusted-root/sigstore-trusted-root-0_p20241010.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="trusted-root.json for dev-python/sigstore"
+HOMEPAGE="https://www.sigstore.dev/"
+SRC_URI="
+	https://dev.gentoo.org/~mgorny/dist/${P}.tar.xz
```

Comment:
Explicit instructions on generating SRC_URI might be nice.

Reply:
Ok, I didn't add instructions per se, but instead made it create a new `.tar.xz` on mismatch. Does that look good?

Reply:
Looks good to me... There are other packages which basically do something the same thing in a live ebuild using `maint_pkg_create` so the idea is quite solid. No real point in doing a live ebuild thing if there's no live sources to checkout. :D

Reply:
wfm